ComposeStarter/app/src/main/res/values-round/strings.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+     Copyright (C) 2021 The Android Open Source Project
+     Licensed under the Apache License, Version 2.0 (the "License");
+     you may not use this file except in compliance with the License.
+     You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+     Unless required by applicable law or agreed to in writing, software
+     distributed under the License is distributed on an "AS IS" BASIS,
+     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+     See the License for the specific language governing permissions and
+     limitations under the License.
+-->
+<resources>
+    <string name="hello_world">From the Round world,\nHello, %1$s!</string>
```

Comment:
How do you feel about highlighting the usage of `LocalConfiguration.current.isScreenRound` for deciding between these strings?

Obviously the qualified resource also works, but could be interesting.

Reply:
Maybe I do that in the more advanced sample so we have one of each?